packages/components/src/dateInput/helpers.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import { setDate, setMonth, setYear } from 'date-fns';
+
+export const composeDateFromStrings = (
+  day: string,
+  month: string,
+  year: string,
+): Date => {
+  let date = new Date();
+  date = setDate(date, parseInt(day, 10));
+  date = setMonth(date, parseInt(month, 10));
+  date = setYear(date, parseInt(year, 10));
+  return date;
```

Comment:
What about something like this:
```
return new Date(parseFloat(year), parseFloat(month), parseFloat(day))
```

Reply:
Didnt know we can do that, will change to that